docs/guides/seccomp.md

```diff
@@ -265,14 +306,14 @@ spec:
   securityContext:
     seccompProfile:
       type: Localhost
-      localhostProfile: operator/seccomp-demo/hello-profile
+      localhostProfile: operator/seccomp-demo-hello-profile
```

Comment:
Mmmm why are you making this change?

Reply:
It was because at one moment I added the `Output` in labels and I did not want to deal with '/' because they are not authorized.
But I do not add it now.